eZ Platform/APIs/Languages/README.md

```diff
@@ -0,0 +1,70 @@
+and make sure to cover needs for future UI needs.
+
+### Use Cases
+The identified new behaviour:
+- As a Developer, I want to get API entities in prioritised language of the given SiteAccess,
+  so I don't need to specify language in API Service calls
+- As a Developer, I want to get API entities in one Translation at a time,
+  so I don't need to specify language or use helpers when using the API entities
+
+##### Related use cases
+- As an Editor, in UI I want to be able to select languages of API entities to view and edit translation independently
+  of the current SiteAccess, _so I don't need to deal with several variants of backend to do that_
+- As an Editor, I want to be able to pick language for translation of the UI itself independently of the
+  current SiteAccess, _so I don't need to deal with several variants of backend to do that_
+- As a Developer, I want to be able to handle translations across the API entities, including being able to delete
+  single translations
+
+#### Existing use cases
+- As a Developer, when enabling on a per entity basis always available language feature, I expect to get entities in
+  it's main language if not translated to language(s) provided by either me or automatically set from SiteAccess.
+
+## Firm
+The feature MUST:
+- Not break BC on the 99% use of the API Repository Services usage:
+  - If developer specified custom language, API should still respect that
+  - If languages is not specified it is assumed we can safely _change_ to return the prioritised one as that is what
+    helpers on top would later pick anyway, so we need to make sure the helpers will still work with only one language.
+- Use languages arguments as a priority list consistently across the API, as is the case with search today
+- Automatically inject the list of SiteAccess defined languages unless developer provided one specifically
+zoom
+
+## Preferred
+The feature SHOULD:
+- Provide a way for the 1% use case to be able to easily adopt to still work:
+  - If languages are not specified and developer inteded to load all languages for further processing, this should still
+    be possible somehow
+- Be consistent across all the different API entities and Services in Repository API
+- **v1:** Features being added as part of this change should be done in v1, and the change in behaviour should be done
+  in v2, allowing v1 users to start to be forward compatible.
+
+## Bonus
+The feature MAY:
+- Give better performance by reducing amount of code needed to deal with languages, reduce the size of entities
+  placed in cache and transferred from database
+
+
+## Open Question:
+In the case where Editor A. selects a given language to edit a given entity, and/or B. language for user interface.
+Should this affect language priority when code calls are made to the API? Hence on what is being displayed?
+
+Clarification:
+A: In this case it is clear language user picks should be specified on calls to API on the entity editor works with,
```

Comment:
I have to say I'm like Bertrand, I don't understand the question. What is exactly the language priority? What is being displayed?
I understand that when editing an entity in a specific language, the underlying communication through the API should indicate the language which the user wants to edit.